pilot/pkg/networking/core/v1alpha3/httproute_test.go

```diff
@@ -55,8 +55,8 @@ func TestGenerateVirtualHostDomains(t *testing.T) {
 			node: &model.Proxy{
 				DNSDomain: "local.campus.net",
 			},
-			want: []string{"foo", "foo.local", "foo.local.campus", "foo.local.campus.net",
-				"foo:80", "foo.local:80", "foo.local.campus:80", "foo.local.campus.net:80"},
+			want: []string{"foo.local", "foo.local.campus", "foo.local.campus.net",
```

Comment:
I am wondering if this case output is right, as this is not even k8s service domain

Reply:
What do you expect here?

Reply:
"foo", "foo:80" "foo.local.campus.net" "foo.local.campus.net:80"

Reply:
Ah yeah I think I agree, as this is splitting the domain